modules/services/cbatticon.nix

```diff
@@ -0,0 +1,106 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.services.cbatticon;
+  package = pkgs.cbatticon;
+  commandLine = concatStringsSep " " (
+    optional
+      (cfg.commandCriticalLevel != null)
+      ("--command-critical-level '${cfg.commandCriticalLevel}'") ++
```

Comment:
Perhaps the command should be wrapped in a shell script? I.e., having something like
```
let
  cmd = pkgs.writeShellScript "cbatticon-critical" cfg.commandCriticalLevel;
in
  "--command-critical-level '${cmd}'"
```
Similarly for `commandLeftClick`.

Then wouldn't have to worry about escaping the string and such.